lib/smart_todo/events/gem_bump.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+require 'bundler'
```

Comment:
If i'm using bundler here, does that mean we should move `bundler` from a `development_dependency` to  a`dependency` to the gemspec?

Reply:
I added a change to enable the gem bundler when this file gets autoloaded so that user not making use of this event don't have to install bundler.